packages/ember-views/lib/system/build-component-template.js

```diff
@@ -43,6 +44,31 @@ export default function buildComponentTemplate({ component, layout, isAngleBrack
   return { createdElement: !!tagName, block: blockToRender };
 }
 
+// Static flag used to see if we can mutate the type attribute on elements. IE8
+// does not support changing the type attribute after an element is inserted in
+// a tree.
+var isTypeAttributeMutable = (function() {
+  var docFragment = document.createDocumentFragment();
+  var mutableInputTypeTextElement = document.createElement('input');
+  mutableInputTypeTextElement.type = 'text';
+  try {
+    docFragment.appendChild(mutableInputTypeTextElement);
+    mutableInputTypeTextElement.setAttribute('type', 'password');
+  } catch (e) {
+    return false;
+  }
+  return true;
+})();
+
+function canChangeTypeAfterRender(attrs) {
+  // This permits testing of the unbound type attr behavior outside of IE8.
+  if (attrs.ie8SafeInput) {
```

Comment:
A [BUGFIX release] should not add new API.  I would be fine with either prevent any changing of type after initial render (this was asserted in some versions of Ember prior to 1.13 anyways)...